### PR TITLE
test: Add team name test cases

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: app/script/auth/module/action/UserAction.test.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ *
+ */
+
+import * as UserAction from './UserAction';
+import * as UserActionCreator from './creator/UserActionCreator';
+import {mockStore} from '../../util/TestUtil';
+
+describe('UserAction', () => {
+  describe('when doing something', () => {
+    it('shows success', done => {
```
Comment (gregor) [inline app/script/auth/module/action/UserAction.test.js]: Weird wording?

File: app/script/auth/page/Root.test.jsx [hunk 2]
```diff
@@ -18,21 +18,37 @@
  */
 
 import React from 'react';
-import {mockStore, renderWithStore} from '../util/TestUtil';
+import {mockStore, mountWithStore} from '../util/TestUtil';
 import Root from './Root';
 
-describe('Root', () => {
-  it('renders the Wire logo', () => {
+describe('when opening the main path', () => {
```
Comment (gregor) [inline app/script/auth/page/Root.test.jsx]: I thought the top level describe should always name the component your are testing.

File: app/script/auth/page/TeamName.test.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    };
+
+    store = mockStore(state);
+    wrapper = mountWithIntl(<TeamName />, store);
+
+    teamNameInput = wrapper.find('[data-uie-name="enter-team-name"]').first();
+    doNextButton = wrapper.find('[data-uie-name="do-next"]').first();
+  });
+
+  it(`doesn\'t show a next button if too few characters are entered`, () => {
```
Comment (gregor) [inline app/script/auth/page/TeamName.test.jsx]: Let's avoid these ` and simply go with `do not` instead

File: app/script/auth/module/action/UserAction.test.js (hunk 1 above)
Comment (bennycode) [reply app/script/auth/module/action/UserAction.test.js]: Weird wording!

**Pull request merged**